src/wizard/rust/profiling-onboarding/rust/2.configure-performance.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Rust
+doc_link: https://docs.sentry.io/platforms/rust/profiling/
+support_level: alpha
+type: language
+---
+
+#### Configure Performance
+
+Sentry’s performance monitoring product has to be enabled in order for Profiling to work. To enable performance monitoring in the SDK:
+
+```rust
+sentry::init((
+    "___DSN___",
+    sentry::ClientOptions {
+        traces_sample_rate: 1.0,
```

Comment:
I believe all the docs should also configure a `release`, right?

Reply:
I didn't know about this requirement for docs. I'll add it to align with the rest then.